test/component/tasks/shellcheck/task.ts

```diff
@@ -68,14 +68,6 @@ suite('ShellCheck Task', () => {
         assert.isTrue(taskLibSetResultStub.calledWithExactly(failedResult, expErrorMessage, true));
     });
 
```

Comment:
The below test was accidentally introduced in a copy pasta error. Removing it accordingly